opentelemetry-etw-logs/examples/basic.rs

```diff
@@ -0,0 +1,39 @@
+//! run with `$ cargo run --example basic --all-features
```

Comment:
can you add instructions on how to use logman or perfview to view the telemetry emitted to etw?

Reply:
On it.